UserInterface.cpp

```diff
@@ -432,7 +431,7 @@ SplashScreenResult displaySplashScreen(const std::wstring& ra3Path, const Langua
 		backgroundBrushWithText = createPatternBrush(background.get());
 	};
 
-	handlers[WM_INITDIALOG] = [&languageData, &icon, &lastTickCount, initializeBrushes](HWND window, WPARAM, LPARAM) {
+	handlers[WM_INITDIALOG] = [=, &languageData, &icon, &lastTickCount, &initializeBrushes](HWND window, WPARAM, LPARAM) {

```

Comment:
Explicit captures was used to make clear which variables has been captured (and which not). 
If you do want to use default capture for readability purposes, then I think it's better to use reference capture default (`&`).

Reply:
This is enforced by MSVC, though I don't know why. It just complains about no default capture mode. See [](https://docs.microsoft.com/zh-cn/cpp/error-messages/compiler-errors-2/compiler-error-c3493)
Actually I didn't checked every capture. In my mind value capture should be used on every variable of pirimitive type? Or we might just keep it with gcc's default behaviour.